apps/datahub/src/app/organization/organization-details/organization-details.component.html

```diff
@@ -0,0 +1,164 @@
+<ng-container *ngrxLet="lastPublishedDatasets$ as lastPublishedDatasets">
+  <div id="organization-details-container" class="flex flex-col gap-[72px]">
+    <div class="h-full container-lg mx-auto px-[25px] w-full">
+      <div class="flex flex-row justify-between gap-[70px]">
+        <!------------------->
+        <!--  LEFT COLUMN  -->
+        <!------------------->
+        <div id="organization-details-left" class="w-2/3 mt-14">
+          <div class="flex flex-col gap-11">
+            <gn-ui-max-lines [maxLines]="2" *ngIf="organization.description">
+              <div>
+                <gn-ui-markdown-parser
+                  data-cy="organizationDescription"
+                  data-test="organizationDescription"
+                  [textContent]="organization.description"
+                ></gn-ui-markdown-parser>
+              </div>
+            </gn-ui-max-lines>
+          </div>
+        </div>
+
+        <!-------------------->
+        <!--  RIGHT COLUMN  -->
+        <!-------------------->
+        <div
+          id="organization-details-right"
+          class="w-[300px] flex flex-col gap-5"
+        >
+          <div
+            data-cy="organizationLogo"
+            class="bg-white border border-[#d8d8d8] w-full h-[185px] rounded-lg p-[30px] -mt-28"
+          >
+            <gn-ui-thumbnail
+              class="relative"
+              [thumbnailUrl]="organization.logoUrl?.href"
+              [fit]="'contain'"
+            >
+            </gn-ui-thumbnail>
+          </div>
+          <a
+            data-cy="organizationDatasetCount"
+            [routerLink]="['/', ROUTER_ROUTE_SEARCH]"
+            [queryParams]="{ publisher: organization.name }"
+          >
+            <gn-ui-figure
+              data-test="organizationDatasetCount"
+              class="py-[37px] pl-[47px] rounded-lg border bg-white mb-5 card-shadow cursor-pointer"
+              [figure]="organization.recordCount"
+              [icon]="'folder_open'"
+              [title]="'catalog.figures.datasets'"
+              [color]="'secondary'"
+            ></gn-ui-figure>
+          </a>
+
+          <a
+            data-cy="organizationEmail"
+            data-test="organizationEmail"
+            [href]="'mailto:' + organization.email"
```

Comment:
The button doesn't work as such. It needs `target="_blank"` to open the user's mailbox. When I tested and clicked on it, nothing happened.

Reply:
It used to work and opened my thunderbird but it doesn't work for me neither now...

Reply:
Found why, it's the gn-ui-button that mess up with the <a>. I remove it and applyed the same style.

Reply:
Okay good! Important thing is that it works :+1: